include/sddf/blk/queue.h

```diff
@@ -132,7 +132,7 @@ static inline bool blk_resp_queue_empty(blk_queue_handle_t *h)
  */
 static inline bool blk_req_queue_full(blk_queue_handle_t *h)
 {
-    return !((h->req_queue->head - h->req_queue->tail + 1) % h->queue_size);
+    return h->req_queue->head - h->req_queue->tail + 1 == h->req_queue->size;
```

Comment:
I assume that blk seems to use head/tail in the opposite way to i2c/network/serial?

Reply:
Yes, I noticed that and made a comment https://github.com/au-ts/sddf/issues/78#issuecomment-2109296110.

Reply:
Maybe @erichchan999 can fix that or I can make another PR later.

Reply:
Yep happy to make a separate PR once this is merged